src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16031,6 +16034,7 @@ namespace UnitarySystems {
             }
 
             elecHeatingPower = state.dataHVACGlobal->DXElecHeatingPower;
+            defrostElecPower = state.dataHVACGlobal->DefrostElecPower;
```

Comment:
Saves `state.dataHVACGlobal->DefrostElecPower` in a local variable. Later this local variable is added to the parent `electricity rate`.